stop_words.txt

```diff
@@ -156,10 +156,13 @@ submodules
 sucre
 tagid
 TAGID
+tagtype
```

Comment:
I don't think this are part of the fixing typos PR, are they?

Reply:
I'm guessing that's a list of words that should be ignored by the tool used to check for typos. Is that correct @dipietroR ? Is it possible to add a comment at the top of this file to indicate what it is / how it works? e.g. how does one check for typos?

Reply:
The file was added for [PodSpelling](https://metacpan.org/pod/Perl::Critic::Policy::Documentation::PodSpelling) policy. Comments should be fine.